### ISSUE TITLE
lncrawl only downloads first 100 chapters from lightnovelpub.com

### ISSUE THREAD

**Issue by swanknight** [dipu-bd/lightnovel-crawler#899]
**Describe the bug**
I tried this with a couple of different LNs from lightnovelpub.com and I got the same result, lncrawl only detects/downloads the first 100 chapters only.

**Let us know**
- Novel URL: https://www.lightnovelpub.com/novel/seoul-stations-necromancer-wn-09041243
- App Version: 2.26.3
- Package: exe
- OS: Windows 10

**Screenshot**
https://i.imgur.com/Re0Bbtc.png

**Comment by SirGryphin** [comment]
I've noticed this as well. It's because there not enough chapters/pages to have a last page button.

![Untitled](https://user-images.githubusercontent.com/36343615/120633286-e18e4000-c461-11eb-8953-5f24609eca5c.png)

This is the piece of code that stop it from working without the `.PagedList-skipToLast` button it doesn't know what to do. I've tried making it use both `.PagedList-skipToLast` and `.PagedList-skipToNext` but it fails to grab all chapters.

I'm not sure who the original author of source was might want to wait for @dipu-bd see if he created the original and if he can modify it to do both.

```
            last_page = soup.select_one('.PagedList-skipToLast a')['href']
            page_count = int(re.findall(r'page=(\d+)', last_page)[0])
        except:
            page_count = 0
        logger.info('Total pages: %d', page_count)
```
